src/usr/console/daemon.C

```diff
@@ -70,6 +72,13 @@ namespace CONSOLE
             Uart::g_device->initialize();
         }
 
+        // Display a banner denoting the hostboot version
+        char banner[256];
+        snprintf(greeting, sizeof(greeting),
```

Comment:
FYI - This doesn't compile, looks like you switched some var names at the last minute.

daemon.C: In function 'void* CONSOLE::consoleDaemon(void*)':
daemon.C:77:18: error: 'greeting' was not declared in this scope
         snprintf(greeting, sizeof(greeting),

I'll fix this up in my local push assuming no other issues.